src/reynir_correct/errfinder.py

```diff
@@ -704,32 +757,40 @@ def VillaFsMeðFallstjórn(
             )
         # In this case, there's no suggested correction
         # and it is likely a wrong annotation causing false positives
-        return dict()
-        #return dict(
+        return None
+        # return dict(
         #    text="Forsetningin '{0}' stýrir {1}falli.".format(
         #        txt.split()[0].lower(), CASE_NAMES[variants]
         #    ),
-        #)
+        # )
 
-    def AðvörunSvigaInnihaldNl(self, txt: str, variants: str, node: Node) -> AnnotationReturn:
-        """ Explanatory noun phrase in a different case than the noun phrase
-        that it explains """
+    def AðvörunSvigaInnihaldNl(
+        self, txt: str, variants: str, node: Node
+    ) -> AnnotationReturn:
```

Comment:
Looks like AnnotationDict would be fine here, instead of AnnotationReturn?

Reply:
Yes, done